plugin/src/main/scala/org/flywaydb/play/PlayInitializer.scala

```diff
@@ -113,6 +113,9 @@ class PlayInitializer @Inject() (implicit app: Application, webCommands: WebComm
     webCommands.addHandler(flywayWebCommand)
 
     for (dbName <- allDatabaseNames) {
+      if (flywayConfigurations(dbName).cleanOnStart) {
```

Comment:
@vjkoskela 
this option is too dangerous for production environment.
it's necessary to check if the current envirionment is test environment here with `Play.isTest`.